inventory/group_vars/all.yml

```diff
@@ -388,6 +388,18 @@ nginx_sites:
         try_files /maintenance.html =503;
       }
 
+      location /cable {
+        {{ nginx_valid_methods }}
```

Comment:
It seems like a good idea that we limit for only the valid methods here. 
There's no comment for `nginx_valid_methods` so I assume it's just general good practice to only open up what's necessary. In that case, I wonder if it could be limited further, because websockets probably don't require all those verbs.

But.. I don't think it matters so would approve this.

Reply:
I've used the same that others locations. I agree, it's probably not ideal. Maybe leave as it for now?